migrator/src/main/java/com/redhat/lightblue/migrator/Controller.java

```diff
@@ -224,15 +224,10 @@ public void run() {
                 MigrationConfiguration[] cfg=getMigrationConfigurations();
                 createControllers(cfg);
                 Breakpoint.checkpoint("Controller:createconfig");
-            } catch (Exception e) {
+                Thread.sleep(30000);
+            } catch (Throwable e) {
```

Comment:
This thread somehow terminates. Any exception thrown here, including InterruptedException, doesn't mean the thread should terminate. It should only terminate when stopped is set.